src/SFA.DAS.PR.Application/Requests/Commands/CreateNewAccountRequest/CreateNewAccountRequestCommandValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+using SFA.DAS.PR.Application.Common.Validators;
+using SFA.DAS.PR.Domain.Entities;
+using SFA.DAS.PR.Domain.Interfaces;
+
+namespace SFA.DAS.PR.Application.Requests.Commands.CreateNewAccountRequest;
+
+public class CreateNewAccountRequestCommandValidator : AbstractValidator<CreateNewAccountRequestCommand>
+{
+    public static readonly string UkprnValidationMessage = "A Ukprn must be provided.";
+    public static readonly string EmployerContactEmailValidationMessage = "EmployerContactEmail must be in the correct email format.";
+
+    public CreateNewAccountRequestCommandValidator(
+        IRequestReadRepository requestReadRepository,
+        IProviderReadRepository providerReadRepository
+    )
+    {
+        RuleFor(x => x.Ukprn)
+            .NotEmpty()
+            .WithMessage(UkprnValidationMessage);
+
+        RuleFor(x => x.Ukprn)
+            .IsValidUkprn(providerReadRepository);
+
+        RuleFor(a => new EmployerPayeRequestObject() { Ukprn = a.Ukprn, EmployerPAYE = a.EmployerPAYE, RequestStatuses = new[] { RequestStatus.New, RequestStatus.Sent } })
+            .ValidateRequest(requestReadRepository);
+
+        RuleFor(a => a.Operations).ValidateOperationCombinations();
+
+        RuleFor(a => a.EmployerContactEmail)
+            .EmailAddress()
```

Comment:
I'm not sure this works properly. I've seen it allow emails with spaces in the front part.  Add such an email in the tests, and if it passes, that's fine.  If it doesn't, you will need to check using regex.